dataclients/kubernetes/kube.go

```diff
@@ -293,6 +297,10 @@ func New(o Options) (*Client, error) {
 		o.AllowedExternalNames = []*regexp.Regexp{regexp.MustCompile(".*")}
 	}
 
+	if algo, err := loadbalancer.AlgorithmFromString(DefaultLoadBalancerAlgorithm); err != nil || algo == loadbalancer.None {
+		o.DefaultLoadBalancerAlgorithm = DefaultLoadBalancerAlgorithm
```

Comment:
this will override the value that the user set to DefaultLoadBalancerAlgorithm in case AlgorithmFromString(DefaultLoadBalancerAlgorithm) is returning an error.
I guess you mean:
```
if algo, err := loadbalancer.AlgorithmFromString(o.DefaultLoadBalancerAlgorithm); err != nil || algo == loadbalancer.None {
o.DefaultLoadBalancerAlgorithm = DefaultLoadBalancerAlgorithm
}